contracts/FutarchyTemplate.sol

```diff
@@ -1,13 +1,129 @@
 pragma solidity 0.4.24;
 
+import "@aragon/templates-shared/contracts/TokenCache.sol";
 import "@aragon/templates-shared/contracts/BaseTemplate.sol";
 
-contract FutarchyTemplate is BaseTemplate {
-  constructor(DAOFactory _daoFactory, ENS _ens, MiniMeTokenFactory _miniMeFactory, IFIFSResolvingRegistrar _aragonID)
-  BaseTemplate(_daoFactory, _ens, _miniMeFactory, _aragonID)
+contract FutarchyTemplate is BaseTemplate, TokenCache {
```

Comment:
where does FutarchyApp even get pulled in??

Reply:
@emaG3m it doesn't yet, will add that in a follow-up. just getting the base stuff in now